.rubocop.yml

```diff
@@ -1,7 +1,9 @@
-require: rubocop-rails
+require:
+  - rubocop-rails
+  - rubocop-rake
 
 AllCops:
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 2.6
```

Comment:
better 2.7+ ?

Reply:
Back when I created this PR, 2.6 was still fine, but it's getting close to also be EOL, so maybe we should also already drop it? :thinking:

Reply:
> so maybe we should also already drop it? 

Yeah, 2.6 will be EOL at the end of this month, 2022-03-31 - so I don't think continuing to support it is reasonable.

Reply:
Ok, I dropped 2.6 and added 3.1 instead (and also fixed that 3.0 stays 3.0 for GH-Actions)